src/Styleguide/Elements/FollowIcon.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react"
+import styled from "styled-components"
+import { themeGet } from "styled-system"
+import Icon from "../../Components/Icon"
+import { Flex } from "./Flex"
+import { Serif } from "@artsy/palette"
+
+interface FollowIconProps {
```

Comment:
Figured we can move this `+ Follow` into an element here. The idea is that it's just a display and then some other component like `ArtistFollow` or whatever will deal with data will actually handle the change in status.